### PR TITLE
bootstrap: stop trashing (DY)LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by ishitatsuyuki**
This generated an ugly error with fakeroot before.

### CODE REVIEW

**Comment by nagisa** [comment]
Should not commit submodule changes.

**Comment by ishitatsuyuki** [comment]
Fixed.

File: src/bootstrap/bootstrap.py [hunk 1]
```diff
@@ -342,8 +342,12 @@ def build_bootstrap(self):
         env = os.environ.copy()
         env["CARGO_TARGET_DIR"] = build_dir
         env["RUSTC"] = self.rustc()
-        env["LD_LIBRARY_PATH"] = os.path.join(self.bin_root(), "lib")
-        env["DYLD_LIBRARY_PATH"] = os.path.join(self.bin_root(), "lib")
+        env["LD_LIBRARY_PATH"] = os.path.join(self.bin_root(), "lib") + \
+                                 (os.pathsep + env["LD_LIBRARY_PATH"]) \
+                                 if "LD_LIBRARY_PATH" in env else ""
+        env["DYLD_LIBRARY_PATH"] = os.path.join(self.bin_root(), "lib") + \
+                                   (os.pathsep + env["DYLD_LIBRARY_PATH"]) \
```
Comment (ranma42) [inline src/bootstrap/bootstrap.py]: Is `os.pathsep` the correct separator here? According to the dyld manpage the paths are separated by ':' (and IIRC the same holds for `LD_LIBRARY_PATH`, too)

Comment (ishitatsuyuki) [inline src/bootstrap/bootstrap.py]: According to [docs](https://docs.python.org/3/library/os.html):
> The character conventionally used by the operating system to separate search path components (as in PATH), such as `':'` for POSIX or `';'` for Windows.

This variable is POSIX only anyway, so it works as intended.

**Pull request merged**